jobserver/management/commands/rap_status.py

```diff
@@ -0,0 +1,41 @@
+"""Management command to update the status of a RAP's Jobs via the RAP API."""
+
+import json
+
+import structlog
+from django.core.management.base import BaseCommand
+
+from jobserver import rap_api
+
+
+class Command(BaseCommand):
+    """Management command to update the status of a RAP's Jobs via the RAP API."""
+
+    help = "Update RAP's jobs status via the RAP API."
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            "rap_ids",
+            nargs="+",
+            type=str,
+            help="IDs of the RAPs to update.",
+        )
+
+    def handle(self, *args, **options):
+        logger = structlog.get_logger(__name__)
+        try:
+            json_response = json.loads(
```

Comment:
rap_api.status() is returning response.json(), which does the json decoding, so I think json.loads() will error here because it's expecting a string

Reply:
ah, you're right. I think this is a problem with the tests which I've unwittingly cribbed from the cancel endpoint - in the tests `response.json()` was returning a bytearray not a dict. In cancel it doesn't matter because it just dumps the value to logging, but I'm actually using it here. I will fix here & then fix the tests for cancel. 

For ref I've just run it outside of the tests, and it fails thusly:
```
$ just manage rap_status ht5xd5jeei4lnwnf d5uenk57whnd4rdc 1234567812345678
$BIN/python manage.py rap_status ht5xd5jeei4lnwnf d5uenk57whnd4rdc 1234567812345678
2025-09-05T09:52:02.555867Z [error    ] the JSON object must be str, bytes or bytearray, not dict
```